packages/local_auth/local_auth/README.md

```diff
@@ -8,9 +8,9 @@ the user.
 On supported devices, this includes authentication with biometrics such as
 fingerprint or facial recognition.
 
-|             | Android   | iOS  |
-|-------------|-----------|------|
-| **Support** | SDK 16+\* | 9.0+ |
+|             | Android   | iOS  | Windows     |
+|-------------|-----------|------|-------------|
+| **Support** | SDK 16+\* | 9.0+ | Windows 10+ |
```

Comment:
I think it would be worth adding a note to the end of the `### Options` section (right after the example of using `biometricOnly` to explain that it's not supported on Windows (maybe with a brief explanation of why).

Reply:
Added.